rpc/proto/mayastor.proto

```diff
@@ -164,6 +164,10 @@ message Nexus {
   // Missing property and empty string are treated the same.
   string device_path = 5;
   uint64 rebuilds = 6;         // total number of rebuild tasks
+  oneof target {
+    string  nbd = 7;
+    string  iscsi = 8;
+  }
```

Comment:
why do we need this? This information is part of URL.

Reply:
Doh! leftover change from when I was using lookup_nexus only and not publish context.
I will remove and the tentacles in rust code as well.